bl-languages/fi_FI.json

```diff
@@ -0,0 +1,242 @@
+	"you-can-modify-the-url-which-identifies":"Voit muokata sivun osoitetta. Enintään 150 merkkiä",
+	"this-field-can-help-describe-the-content": "Kirjoita tähän kuvaus sivun sisällöstä. Kuvaus näkyy hakutuloksessa esim. Googlessa. Enintään 150 merkkiä.",
+
+	"delete-the-user-and-all-its-posts":"Poista käyttäjä, sekä kaikki sen blogikirjoitukset",
+	"delete-the-user-and-associate-its-posts-to-admin-user": "Poista käyttäjä, ja siirrä sen blogikirjoitukset ylläpitäjän nimiin",
+	"read-more": "Lue lisää...",
+	"show-blog": "Näytä blogi",
+	"default-home-page": "Sivuston etusivu",
+	"version": "Versio",
+	"there-are-no-drafts": "Ei luonnoksia.",
+	"create-a-new-article-for-your-blog":"Kirjoita blogiisi.",
+	"create-a-new-page-for-your-website":"Luo uusi sivu sivustollesi.",
+	"invite-a-friend-to-collaborate-on-your-website":"Tee ystävällesi omat tunnukset, niin hänkin voi kirjoittaa sivustollesi",
+	"change-your-language-and-region-settings":"Muuta kielen ja alueen asetuksia.",
+	"language-and-timezone":"Kieli ja aikavyöhyke",
+	"author": "Tekijä",
+	"start-here": "Aloita tästä",
+	"install-theme": "Asenna teema",
+	"first-post": "Ensimmäinen blogikirjoitus",
+	"congratulations-you-have-successfully-installed-your-bludit": "Onneksi olkoon! **Bludit** on nyt asennettu!",
+	"whats-next": "Mitä seuraavaksi?",
+
+	"follow-bludit-on": "Seuraa Bludit:ia",
+	"visit-the-support-forum": "Vieraile [foorumilla](https://forum.bludit.com) jos tarvitse tukea",
+	"read-the-documentation-for-more-information": "[Dokumentaatiosta](https://docs.bludit.com) löydät paljon lisää tietoa",
+	"share-with-your-friends-and-enjoy": "Jaa ystäviesi kanssa ja nauti!",
+	"the-page-has-not-been-found": "Etsimääsi sivua ei löydy.",
+	"error": "Virhe",
+	"bludit-installer": "Bludit Installer",
+	"welcome-to-the-bludit-installer": "Tervetuloa asentamaan Bludit!",
+	"complete-the-form-choose-a-password-for-the-username-admin": "Täytä lomake, valitse salasana käyttäjälle « admin » (sivuston ylläpitäjä)",
+	"password-visible-field": "Salasana, näkyvä kenttä!",
+	"install": "Asenna",
+	"choose-your-language": "Valitse kielesi",
+	"next": "Seuraava",
+	"the-password-field-is-empty": "Salasanakenttä on tyhjä",
+	"your-email-address-is-invalid":"Sähköpostiosoitteesi on vääränlainen.",
+	"proceed-anyway": "Jatka silti!",
+	"drafts":"Luonnokset",
+	"ip-address-has-been-blocked": "IP osoitteesi on estetty.",
+	"try-again-in-a-few-minutes": "Yritä uudelleen muutaman minuutin päästä.",
+	"date": "Päivämäärä",
+
+	"scheduled": "Ajastettu",
+	"publish": "Julkaise",
+	"please-check-your-theme-configuration": "Tarkista teeman asetukset.",
+	"plugin-label": "Lisäosan nimi",
+	"enabled": "Käytössä",
+	"disabled": "Poissa käytöstä",
+	"cli-mode": "CLI-tila",
+	"command-line-mode": "Komentorivitila",
+	"enable-the-command-line-mode-if-you-add-edit": "Ota käyttöön komentorivitila, jos lisää, muokkaat tai poistat sivuja tai blogikirjoituksia suoraan tiedostojärjestemästä",
```

Comment:
"enable-the-command-line-mode-if-you-add-edit": "Ota käyttöön komentorivitila, jos lisäät, muokkaat tai poistat sivuja tai blogikirjoituksia suoraan tiedostojärjestemästä",